### DISCUSSION TITLE
I'm an idiot just starting PMM using Docker

### DISCUSSION THREAD

**Discussion by 13rambo13** [meisnate12/Plex-Meta-Manager-Configs#593] [Q&A]
I am trying to guess my PMM path because i HAVE NO IDEA WHAT i AM DOING. i FOLLOWED TTHE INSTRUCTIONS AND MADE IT FINE UNTIL i NEED TO RUN THE SCRIPT FOR THE FIRST TIME. i AM SO LOST HERE IS WHAT i HAVE TRIED

PS C:\Users\crsor> docker run --rm -it -v C:Users/crsor/Plex-Meta-Manager/config:rw meisnate12/plex-meta-manager --run
docker: Error response from daemon: The system cannot find the path specified.
See 'docker run --help'.
PS C:\Users\crsor> docker run --rm -it -v C:Users/crsor/Plex-Meta-Manager/config:rw meisnate12/plex-meta-manager --run
docker: Error response from daemon: The system cannot find the path specified.
See 'docker run --help'.
PS C:\Users\crsor> docker run --rm -it -v C:Users/crsor/Plex-Meta-Manager/config/config.yml/config:rw meisnate12/plex-me
ta-manager --run
docker: Error response from daemon: The system cannot find the path specified.
See 'docker run --help'.
PS C:\Users\crsor>

**Comment by cpt-kuesel** [comment]
you are missing a slash between the drive letter and the rest of the path
try:
`docker run --rm -it -v C:/Users/crsor/Plex-Meta-Manager/config:rw meisnate12/plex-meta-manager --run`

**Comment by 13rambo13** [comment]
Thanks I appreciate it. I actually just got it to work finally but now am
getting a yaml error when I run the script. This process is beyond
frustrating.
